examples/v2/project_creation/project.py

```diff
@@ -74,6 +74,33 @@ def GenerateConfig(context):
               context.properties['concurrent_api_activation']
       }
   }, {
+      'name': 'get-iam-policy',
+      'action': 'gcp-types/cloudresourcemanager-v1:cloudresourcemanager.projects.getIamPolicy',
+      'properties': {
+        'resource': project_id,
+      },
+      'metadata': {
+        'dependsOn': [ApiResourceName(
+            project_id, 'deploymentmanager.googleapis.com')]
+      }
+  }, {
+   # Add the service account that deployment manager will use in this project
+   # as owner so it can set IAM policies on resources
+      'name': 'patch-iam-policy',
```

Comment:
Should we guard this behind a property ? Not all users would want their project robot to be an owner.